Run native gbe_bin_generater to compile built-in kernels
---
 backend/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/backend/CMakeLists.txt b/backend/CMakeLists.txt
index 915d60f..c3041bc 100644
--- a/backend/CMakeLists.txt
+++ b/backend/CMakeLists.txt
@@ -58,6 +58,6 @@ set (GBE_BIN_GENERATER
      PARENT_SCOPE)
 else (USE_STANDALONE_GBE_COMPILER STREQUAL "true")
 set (GBE_BIN_GENERATER
-     ${GBE_BIN_GENERATER} LD_LIBRARY_PATH=${CMAKE_CURRENT_BINARY_DIR}/src ${CMAKE_CURRENT_BINARY_DIR}/src/gbe_bin_generater
+     ${GBE_BIN_GENERATER} ${STANDALONE_GBE_COMPILER_DIR}/gbe_bin_generater
      PARENT_SCOPE)
 endif (USE_STANDALONE_GBE_COMPILER STREQUAL "true")
-- 
2.5.5

